lib/onigumo.ex

```diff
@@ -13,17 +13,13 @@ defmodule Onigumo do
   end
 
   def download(http_client, dir_path) do
+    file_path = Path.join(dir_path, @output_file_name)
```

Comment:
I don’t really like this. But where else should this belong?

Reply:
```suggestion
    output = Path.join(dir_path, @output_file_name)
```

Maybe ?

Reply:
This was not about the name, but about the location. But there is no better place for now when the name is not dynamic. It does not make sense to generate the same filename for every URL.